src/main/java/com/ontotext/kafka/util/PropertiesUtil.java

```diff
@@ -27,7 +27,7 @@
 
```

Comment:
I don't see the changes where we log the excepliton when validating the /versions url. Did you skip those on purpose?

Reply:
Nope, I missed logging the exception by accident.